api4/usage.go

```diff
@@ -30,3 +32,20 @@ func getPostsUsage(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	w.Write(json)
 }
+
+func getStorageUsage(c *Context, w http.ResponseWriter, r *http.Request) {
+	usage, appErr := c.App.GetStorageUsage()
+	if appErr != nil {
+		c.Err = model.NewAppError("Api4.getPostsUsage", "app.post.analytics_posts_count.app_error", nil, appErr.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	usage = utils.RoundOffToZeroes(float64(usage))
```

Comment:
Rounding off in the api layer instead of the app layer.

Reply:
The purpose of rounding here most likely is to actually hide the real figures from the api consumers.